tests/integration/cassettes/test_gerrit_test_00_review.json

```diff
@@ -0,0 +1,4 @@
+{
+  "http_interactions": [],
+  "recorded_with": "betamax/0.5.1"
+}
```

Comment:
Can we add a EOL of EOF here, or does that bugger up betamax?

Reply:
I am not sure about this. This file was generated automatically by betamax and other cassettes don't have EOL too.

Reply:
Try it.  Manually edit the file after generation.  It may not write good JSON, but it will use the standard JSON loader, which can handle the proper EOF marker.

Worst than can happen is it doesn work and we raise a bug upstream.

Reply:
I added EOL and it works fine

Reply:
well, don't worry about that, anyway those files will be overwritten at a later stage with actual data. About the proper JSON format, that will happen when we'll update to the latest betamax, so don't worry about that! 😉

Currently betamax is pinned to 0.5.1, because with a commit that will update only the tests, because it's painful to see all your tests rewritten because of an update of betamax, and not of your code.